web/templates/navbar.hbs

```diff
@@ -33,7 +33,7 @@
                 <a href="/register/"  class="signup_button">
                     {{t 'Sign up' }}
                 </a>
-                <a class="login_button">
+                <a href="/login/" class="login_button">
                     {{t 'Log in' }}
```

Comment:
You can use `tabindex` instead of `href` here.

Reply:
Updated